test/suites/smoke-test-common-solo/test-ethereum-token-transfers.ts

```diff
@@ -171,7 +171,23 @@ describeSuite({
                                 eventLog.topics
                             );
 
-                            const versioned = api.registry.createType("VersionedXcmMessage", decodedEvent.payload);
+                            let versioned;
+                            try {
+                                versioned = api.registry.createType("VersionedXcmMessage", decodedEvent.payload);
```

Comment:
This way we always hide errors decoding message. I would prefer to check for the magic bytes of the message, and decide if its a validator update or a VersionedXcmMessage based on that.

```rust
pub const MAGIC_BYTES: [u8; 4] = [112, 21, 0, 56];
```

Reply:
me too, but that requiers a bunch of work that can be converted into another issue. I dont want alerts right now to mess up with us

Reply:
or if you want I can test only the first 4 bytes, if that sounds good. sorry I though you meant decoding the entire message

Reply:
I can add a quick thing that @tmpolaczyk mentioned, because I like it better than just try/catch thing and we can test it on both blocks.